tensordict/tensordict.py

```diff
@@ -89,6 +88,38 @@
 _STR_MIXED_INDEX_ERROR = "Received a mixed string-non string index. Only string-only or string-free indices are supported."
 
 
+class _TensorDictKeysView:
+    """
+    Wrapper class that enables richer behaviour of `key in tensordict.keys()`
```

Comment:
Maybe add something along the lines of "_TensorDictKeysView is returned when accessing tensordict.keys() and holds a reference to the original tensordict. This class enable us to modify the behaviour of key in tensordict.keys() and use tuples for nested key lookups. For example ... " (taken from the description of the PR)

Reply:
I'll be doing some more work on this today, so I'm happy to expand the docstring, I think that's a good idea.